CalibTracker/SiPixelESProducers/interface/SiPixelGainCalibrationForHLTLayout.h

```diff
@@ -0,0 +1,40 @@
+#ifndef DataFormats_SiPixelGainCalibrationForHLTSoA_interface_SiPixelGainCalibrationForHLTLayout_h
+#define DataFormats_SiPixelGainCalibrationForHLTSoA_interface_SiPixelGainCalibrationForHLTLayout_h
+
+#include <array>
+#include "DataFormats/SoATemplate/interface/SoALayout.h"
+#include "DataFormats/TrackerCommon/interface/SimplePixelTopology.h"
+
+namespace siPixelGainsSoA {
+  struct DecodingStructure {
+    uint8_t gain;
+    uint8_t ped;
+  };
+
+  using Range = std::pair<uint32_t, uint32_t>;
+  using RangeAndCols = std::array<std::pair<Range, int>, phase1PixelTopology::numberOfModules>;
+
+}  // namespace siPixelGainsSoA
+
+GENERATE_SOA_LAYOUT(SiPixelGainCalibrationForHLTLayout,
```

Comment:
The SoA layout and the corresponding PortableCollections correspond to data formats, and I think they should be placed as such. On a cursory look `CondFormats/SiPixelObjects` would look like a plausible candidate (even if none of these are put in the CondDB).

Reply:
@makortel should this also be done for `SiPixelMapping`? The SoA layout and the PortableCollections are in this same package.

Reply:
Yes, I think all layout and PortableCollection definitions currently in `CalibTracker/SiPixelESProducers/interface/` should be moved to a data format package.